dot/network/light_test.go

```diff
@@ -0,0 +1,103 @@
+package network
+
+import (
+	"testing"
+
+	"github.com/ChainSafe/gossamer/lib/utils"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDecodeLightMessage(t *testing.T) {
+	s := &Service{
+		lightRequest: make(map[peer.ID]struct{}),
+	}
+
+	testPeer := peer.ID("noot")
+
+	testLightRequest := NewLightRequest()
```

Comment:
Do we have tests for invalid request parsing?

Reply:
I don't think so, I also didn't write this, I just copied it from service_test.go into a new file